resolver/src/resolver/resolve-dependencies/macro-node-to-definition.ts

```diff
@@ -21,11 +21,20 @@ export function macroNodeToDefinition<T>(
       editorComponentBundleContent: "",
     } as MacroEditorConfigCustomDefinition,
   };
-  const editorComponentPath = join(
-    importPath,
+  const BundlePathPlusOne = join(
     "..",
     macro.editorConfig.editorComponentBundlePath,
   );
+  let editorComponentPath = join(
+    importPath,
+    BundlePathPlusOne,
+  );
+  if (!existsSync(editorComponentPath)) {
```

Comment:
let's add a comment here on why we're doing it. something along
`// fallback for backwards compatibility (see issue #120)`